hack/jenkins/upload-to-gcs.sh

```diff
@@ -46,8 +46,22 @@ if [[ ! ${JENKINS_UPLOAD_TO_GCS:-y} =~ ^[yY]$ ]]; then
   exit 0
 fi
 
+GCS_SUBDIR=""
+readonly remote_git_repo=$(git config --get remote.remote.url | sed 's:.*github.com/::' || true)
```

Comment:
I guess I was hoping to get this info from the environment. This line feels super brittle.

1. It will be empty without a git checkout, so we'll default to empty `GCS_SUBDIR`, even if we add a job that tests another repo without checking it out. This isn't a deal-breaker, just something we have to keep in mind.
1. If the remote url ends with ".git", what happens?
1. On my machine it's called "remote.upstream", but this assumes it's "remote.remote". On Jenkins we set that, but I can totally see us changing that at some point and breaking this.

I was hoping to get this info from the environment.